Dockerfile

```diff
@@ -1,49 +1,64 @@
-FROM	debian:wheezy
-
+FROM       node:6.9.0-slim
 MAINTAINER Automattic
 
-WORKDIR /calypso
-
-RUN     mkdir -p /tmp
-COPY    ./env-config.sh /tmp/
-RUN     bash /tmp/env-config.sh
-RUN     apt-get -y update && apt-get -y install \
-          wget \
-          git \
-          python \
-          make \
-          build-essential
-
-ENV NODE_VERSION 6.9.0
-
-RUN     wget https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz && \
-          tar -zxf node-v$NODE_VERSION-linux-x64.tar.gz -C /usr/local && \
-          ln -sf /usr/local/node-v$NODE_VERSION-linux-x64 /usr/local/node && \
-          ln -sf /usr/local/node/bin/npm /usr/local/bin/ && \
-          ln -sf /usr/local/node/bin/node /usr/local/bin/ && \
-          rm node-v$NODE_VERSION-linux-x64.tar.gz
+WORKDIR    /calypso
 
-# npmrc is created by env-config.sh. For local testing, an empty one is generated
-RUN     touch /usr/local/etc/npmrc && \
-          mkdir /usr/local/node/etc && \
-          cp /usr/local/etc/npmrc /usr/local/node/etc/npmrc
 
-ENV     NODE_PATH /calypso/server:/calypso/client
+ENV        NODE_PATH=/calypso/server:/calypso/client
```

Comment:
For anyone reading the `KEY=VALUE` syntax can be used to set multiple env variables. https://docs.docker.com/engine/reference/builder/#env